hack/install-argocd-application.sh

```diff
@@ -1,5 +1,8 @@
-#/bin/sh
+#!/bin/bash
 set -e
 
-../go-mod-hack.sh v1.20.1
+# root=$(dirname "${BASH_SOURCE[0]}")
+
+bash go-mod-hack.sh v1.20.1
 go get github.com/argoproj/argo-cd@a4ee25b
+touch hack/.hack.argocd.installed
```

Comment:
Do we need to `.gitignore` this file?

Reply:
yes, it is just a flag file to make sure we don't have to go through installing the go-mod-hack over and over again.